src/types.h

```diff
@@ -717,6 +717,8 @@ enum class GlobalVar {
   MAX_CPU_ID,
   // Scratch buffer used for format strings to avoid BPF stack allocation limits
```

Comment:
nit: can probably combine these two comments so you don't have to keep repeating it